src/_pytest/_code/code.py

```diff
@@ -876,6 +876,9 @@ def __str__(self):
     def __repr__(self):
         return "<{} instance at {:0x}>".format(self.__class__, id(self))
 
+    def toterminal(self, tw):
```

Comment:
`tw` can't be typed because it's in `py`, but can add `-> None` here (and in all the subclasses if you feel like it).